gradboost_pv/inference/data_feeds.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+from typing import Iterator
+
+import numpy as np
+import xarray as xr
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@dataclass
+class DataInput:
+    nwp: xr.Dataset
+    gsp: xr.Dataset
+    datetime_utc_at_read: np.datetime64
```

Comment:
what does at read mean?

Reply:
will rename to forecast_intitation_datetime_utc